docs/zephyr_01_high_level_requirements.sdoc

```diff
@@ -265,10 +265,13 @@ TYPE: High Level
 COMPONENT: Thread Synchronization
 TITLE: Mutex
 STATEMENT: >>>
-Zephyr shall provide an interface for managing communcation between threads.
+The system shall provide a mutex synchronization primitive that implement a traditional reentrant mutex allowing multiple threads to safely share an
```

Comment:
I would remove wording traditional, it may add confusion because of the interpretation of traditional.

Reply:
done